sdk-blueprints/update-schema.sh

```diff
@@ -0,0 +1,22 @@
+# Internal script for debugging SDK generation locally.
+# Run from the root of the openapi-clients repository.
+
+# Define directory where projects are stored
+projects_dir="$HOME/projects"
```

Comment:
Consider this for parameterization as well (optional `$2` with default as the current value)

Reply:
Added projects_dir as an argument with default value.